packages/connect-core/src/utils/transactions.ts

```diff
@@ -1,52 +1,13 @@
 import { ethers } from 'ethers'
 
 import { erc20ABI, forwarderAbi, forwarderFeeAbi } from './abis'
-import { isFullMethodSignature } from './app'
 import App from '../entities/App'
+import Transaction from '../entities/Transaction'
 
-export interface Transaction {
-  data: string
-  from?: string
-  to: string
-}
-
-export interface TransactionWithTokenData extends Transaction {
-  token: {
-    address: string
-    value: string
-    spender: string
-  }
-}
-
-export async function createDirectTransaction(
-  sender: string,
-  destination: string,
-  methodAbiFragment: ethers.utils.FunctionFragment,
-  params: any[]
-): Promise<Transaction> {
-  let transactionOptions = {}
-
-  // If an extra parameter has been provided, it is the transaction options if it is an object
```

Comment:
We don't longer need to support the transaction options as overrides as we did on aragonAPI. Now, this is the responsibility of the users of Connect when they sign the transaction we provide.